src/zkevm_specs/evm_circuit/execution/callop.py

```diff
@@ -44,9 +48,13 @@ def callop(instruction: Instruction):
     # - caller_address = parent_caller_address
     #
     callee_address = instruction.select(
-        is_callcode + is_delegatecall, caller_address, call.callee_address
+        is_callcode + is_delegatecall, ctx_caller_address, call.callee_address
+    )
+    callee_address_word = instruction.address_to_word(callee_address)
+    caller_address_word = instruction.select(
```

Comment:
I see. Because just 2 line above there are `callee_address` handling. So im just wondering why processing  for `callee_address` vs `caller_address` are in different way. After your fix I think all clear now :)